docs/CONTRIBUTING.md

```diff
@@ -26,15 +26,49 @@ Alternative methods if you don't like Github (also fully supported):
 It is likely however once a patch is ready to apply a maintainer
 will push it to a github PR, and merge via Homu.
 
-Commit message style
---------------------
+## [Commit message style](#commit-message-style)
```

Comment:
We don't have to "linkify" each header like this; GitHub already has a little "link" icon that pops up when you mouse over a header.

Reply:
Okay... noted. This was just in case, that we would make a table of contents on the top of this markdown file